src/clients/http_client/http_client.ts

```diff
@@ -182,11 +188,25 @@ export class HttpClient {
           response.headers &&
           response.headers.has('X-Shopify-API-Deprecated-Reason')
         ) {
-          const deprecation = {
+          const deprecation: DeprecationInterface = {
             message: response.headers.get('X-Shopify-API-Deprecated-Reason'),
             path: url,
           };
 
+          if (options.body) {
+            if (typeof options.body === 'string') {
```

Comment:
Is there really a need for this since https://github.com/Shopify/shopify-api-node/pull/426/files#diff-1d5aeaa437fd851787c18a3ca142885c050d28a290937f6e4d5ca8afea9d1ccfR99-R112 essentially guarantees it will be a string?

Reply:
I had to do this because of the typing of `options.body`, which can be a few different things.